static/temp/foods.json

```diff
@@ -0,0 +1,250 @@
+[
+  {
+    "name": "Banana ripe, slightly ripe, raw",
+    "categoryName": "Fruit and Fruit Juices",
+    "categoryIcon": "EggAltOutlinedIcon",
```

Comment:
We should have 1 object for category such as 

```text
"category": {
    "name": "Fruit and Fruit Juices",
    "icon": "EggAltOutlinedIcon"
}
```

Reply:
done